interface/billing/sl_eob_search.php

```diff
@@ -36,6 +36,7 @@
 require_once "$srcdir/user.inc";
 
```

Comment:
Hi, Anyway, while you are here, make the invoice search default to open in search mode. Nine times out of ten that will be the mode most used and matches the primary purpose of the feature.